server/app/mutations/grid_services/common.rb

```diff
@@ -172,7 +171,7 @@ def common_validations
           end
           string :entrypoint
           array :env do
-            string
+            string matches: /\A\S+(?<!\\)=.*/
```

Comment:
If it's used in multiple places, a constant would make sense. Still IMO it deserves some explanatory comment what it filters. :)